### PR TITLE
Update sum.test.ts

### PR DESCRIPTION
**Pull request by mohamedamer103**
Update sum.test.ts

### CODE REVIEW

**Comment by MohamedTahaAmer** [comment]
Test sending messages

File: src/sum.test.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { sum } from "./sum"
 
 describe("#sum", () => {
 	it("should add two numbers", () => {
-		expect(sum(1, 2)).toBe(3)
+		expect(sum(1, 3)).toBe(4)
```
Comment (MohamedTahaAmer) [inline src/sum.test.ts:12]: Test code comment

**Review by MohamedTahaAmer — COMMENTED** [review]
test review finish message

File: src/sum.test.ts (hunk 1 above)
Comment (mohamedamer103) [reply src/sum.test.ts:12]: Test received from the other end

**Review by mohamedamer103 — COMMENTED** [review]
other end review finish

File: src/sum.test.ts (hunk 1 above)
Comment (mohamedamer103) [reply src/sum.test.ts:12]: Yes we can, no need for another review

**Pull request merged**